### ISSUE TITLE
Use of multiple GitHub organizations is harmful

### ISSUE THREAD

**Issue by per1234** [arduino/Arduino#8245]
Arduino's repositories are now spread across at least four organizations:
- [arduino](https://github.com/arduino)
- [arduino-libraries](https://github.com/arduino-libraries)
- [bcmi-labs](https://github.com/bcmi-labs)
- [vidor-libraries](https://github.com/vidor-libraries)

GitHub has recently added the extremely useful feature of being able to [transfer issues between repositories](https://help.github.com/articles/transferring-an-issue-to-another-repository/). However, this is only allowed for repositories within an organization or owner.

I'm not sure of the feasibility of merging the existing organizations, but it would be a good idea to avoid creating any more in the future.

**Comment by facchinm** [comment]
I agree with you, the new feature is super cool and would have saved a lot of time when switching to arduino-builder some years ago :slightly_smiling_face: 
`arduino-libraries` repo is there to keep some sanity and to have more flexibility in the organization members and permissions. `vidor-libraries` repos could probably be moved to `arduino` organization with no fuss but I don't think there will be many misplaced issues there :wink:

**Comment by cmaglie** [comment]
> However, this is only allowed for repositories within an organization or owner

Hopefully this limit will be removed in the future, it would be super-useful to have the possibility to move issues between different owners / organizations.
